src/CONST.ts

```diff
@@ -5087,7 +5087,7 @@ const CONST = {
             video: {
                 url: `${CLOUDFRONT_URL}/videos/guided-setup-manage-team-v2.mp4`,
                 thumbnailUrl: `${CLOUDFRONT_URL}/images/guided-setup-manage-team.jpg`,
-                duration: 55,
+                duration: 24,
```

Comment:
@Krishna2323 Why are updating durations of unrelated changes? And I think this duration doesn't do anything as far I saw.

Reply:
@Krishna2323 But I still don't think we should update for other videos which are not part of this change. Can we update the duration for the one video alone. Maybe in future we can update others if required. Just wanted to avoid unrelated changes.